### DISCUSSION TITLE
播放合集/收藏夹此类多P视频时 倒序播放

### DISCUSSION THREAD

**Discussion by zhongfly** [the1812/Bilibili-Evolved#1930] [功能建议 / Ideas]
主要原因在于播放的顺序是最新在上，播放时从新到旧，而不少UP发布合集视频（系列视频）时都是1最早发布，再发布2、3、4……这导致播放时实际是逆序的，从后往前播放。
希望能够加入逆序播放
![image](https://user-images.githubusercontent.com/11155705/121482630-e7ef5f80-c9ff-11eb-94bf-cb3a4a6c7269.png)

**Comment by the1812** [comment]
只是这种合集不支持, 收藏夹和稍后再看都是可以倒着的
<img width="300" src="https://user-images.githubusercontent.com/26504152/121482963-3ef53480-ca00-11eb-83bb-89291a1870c6.png">

**Reply by zhongfly** [reply]
那么有可能支持合集吗？

**Reply by the1812** [reply]
不太可能, 你还是想办法把他们加到某个收藏夹里好些

**Comment by Geeyun-JY3** [comment]
我感觉是 B 站的问题……这个功能就应该由 B 站来做的。收藏夹和稍后再看都做了，UP 主这种合集也应该做的。

**Comment by ozerotwoo** [comment]
B站到现在也没改 太难受了 明明手机端就有的功能 为什么网页端一直不做 真不理解；求加一个这个功能吧 等B站自己做等到猴年马月了
